lexicons/com/atproto/admin/defs.json

```diff
@@ -51,6 +58,14 @@
         "cid": { "type": "string", "format": "cid" },
         "recordUri": { "type": "string", "format": "at-uri" }
       }
+    },
+    "threatSignature": {
+      "type": "object",
+      "required": ["property", "value"],
+      "properties": {
+        "property": { "type": "string" },
```

Comment:
The way I think of it, the threat signature has multiple properties, each of which has a name and a value.  So I think of this as a `name`.  But it's truly a bikeshed, all good either way.

Reply:
hmmm yeah i buy it 👍

Reply:
yeah I see where you're coming from, but we already have stuff in entryway codebase around this naming convention and this isn't so far off that I wanna do DB migrations for it 😅